manifests/base/kustomization.yaml

```diff
@@ -403,6 +406,27 @@ vars:
       apiVersion: v1
     fieldref:
       fieldpath: data.odh-codeserver-notebook-image-commit-n-1
+  - name: jupyter-rocm-minimal-notebook-image-commit-n
```

Comment:
You have added only the `*-commit-n` here you will need to do the same for `image-n`